server/src/pages/shot/view.js

```diff
@@ -347,14 +346,13 @@ class Body extends React.Component {
     let editButton;
     const highlight = this.state.highlightEditButton ? <div className="edit-highlight" onClick={ this.onClickEdit.bind(this) } onMouseOver={ this.onMouseOverHighlight.bind(this) } onMouseOut={ this.onMouseOutHighlight.bind(this) }></div> : null;
 
-    if (this.props.isFxaAuthenticated) {
-      const activeFavClass = this.props.expireTime ? "" : "is-fav";
-      favoriteShotButton = <div className="fav-wrapper"><button
-        className={`button favorite ${activeFavClass}`}
-        title="Favorite this shot"
-        onClick={ this.onClickFavorite.bind(this) }
-      /></div>;
-    }
+    const activeFavClass = this.props.expireTime ? "" : "is-fav";
+    const shouldShow = this.props.isFxaAuthenticated ? "" : "hidden";
+    favoriteShotButton = <div className="fav-wrapper"><button
+      className={`button favorite ${shouldShow} ${activeFavClass}`}
+      title="Favorite this shot"
```

Comment:
Do you mind adding the l10n for this title attribute that was missed in the original patch?